specials/templates/specials/special_detail.html

```
@@ -61,29 +61,41 @@
+          {% if audience_choices or place_choices %}
           <div class="select-group">
+            {% if audience_choices %}
             <div class="select-group__item">
-              <select name="{{ target_audience_filter_name }}" aria-label="{% translate 'Busca por público' %}">
-                <option disabled {% if not target_audience_filter_value %}selected{% endif %} value="">{% translate 'Público' %}</option>
-                {% for value, label in target_audience_choices %}
-                  <option value="{{ value }}"{% if value == target_audience_filter_value %} selected{% endif %}>{{ label }}</option>
+              <select name="{{ audience_filter_name }}" aria-label="{% translate 'Busca por público' %}">
+                <option disabled {% if not filters.audience %}selected{% endif %} value="">{% translate 'Público' %}</option>
+                {% for value, label in audience_choices %}
+                  <option value="{{ value }}"{% if value == filters.audience %} selected{% endif %}>{{ label }}</option>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)